src/bctklib/smart-contract/TestApplicationEngine.cs

```diff
@@ -200,19 +200,6 @@ public override VMState Execute()
             return base.Execute();
         }
 
-        protected override void LoadContext(ExecutionContext context)
```

Comment:
Any reason why you remove this?

Reply:
The ApplicationEngine class removes the LoadContext method.

Reply:
I fixed this already https://github.com/neo-project/neo/pull/3243